project1/application.py

```diff
@@ -43,7 +42,7 @@ def register():
         name= request.form.get("username")
         password = request.form.get("password")
         email = request.form.get("email-id")
-        register = database(name= name,email=email,datetime = datetime.now())
+        register = database(name= name,email=email,password = password, datetime = datetime.now())
```

Comment:
you declared datetime as string,are you able to insert values into your table.

Reply:
![image](https://user-images.githubusercontent.com/55687162/79841054-7bb53500-83d4-11ea-9c23-0b23366f55a9.png)

earlier I did not add password mam, later i have added and made commit..this is the screenshot after adding password mam

Reply:
ok fine.